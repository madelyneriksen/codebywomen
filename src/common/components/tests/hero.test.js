import React from 'react';
import renderer from 'react-test-renderer';
import Hero from '../hero';


describe("Hero", () => {
  it("Matches Snapshot", () => {
    const data = {
      title: "The Zen of Programming",
      subtitle: "by Ali Spittel",
    }
    const tree = renderer.create(<Hero props={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
