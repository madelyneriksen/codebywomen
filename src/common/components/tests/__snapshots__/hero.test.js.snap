// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Matches Snapshot 1`] = `
<div
  className="pv5 flex items-center justify-center flex-column bg-lightest-blue"
>
  <h1
    className="f2 f1-ns accent-font navy lh-copy tc"
  />
</div>
`;
